### PR TITLE
fix: output mapping webalize

### LINKED ISSUE #1662: OM - invalid bucket name
when bucket name is not valid, there is no indication that it is the reason why OM cannot be created:

e.g. http://youtu.be/KRobJr3FCyE?hd=1 this is highly confusing

possible solutions:
- highlight invalid bucket name?
- don't juggle with dots?

### PR DESCRIPTION
**Pull request by ondrejhlavacek**
Fixes #1662 

Current

image

Proposed

image

For both buckets and tables

### CODE REVIEW

File: src/scripts/react/common/DestinationTableSelector.jsx [hunk 1]
```diff
@@ -66,27 +63,29 @@ export default React.createClass({
     );
     const bucketSelect = (
       <Select.Creatable
-        promptTextCreator={label => `Create new bucket ${label.startsWith('c-') ? '' : 'c-'}${label}`}
+        promptTextCreator={label => label}
```
Comment (ujovlado) [inline src/scripts/react/common/DestinationTableSelector.jsx]: Toto ma vlastne vyznam, ked ta fn vracia to co dostala?

File: src/scripts/react/common/DestinationTableSelector.jsx [hunk 2]
```diff
@@ -66,27 +63,29 @@ export default React.createClass({
...
         options={this.prepareBucketsOptions().toJS()}
         autosize={false}
+        newOptionCreator={this.selectBucketOptionCreator}
       />
     );
 
     const tableSelect = (
       <Select.Creatable
-        promptTextCreator={label => `Create new table ${label}`}
+        promptTextCreator={label => label}
```
Comment (ujovlado) [inline src/scripts/react/common/DestinationTableSelector.jsx]: detto

File: src/scripts/react/common/DestinationTableSelector.jsx (hunk 1 above)
Comment (ondrejhlavacek) [reply src/scripts/react/common/DestinationTableSelector.jsx]: workaround pro https://github.com/JedWatson/react-select/issues/2229

File: src/scripts/react/common/DestinationTableSelector.jsx (hunk 2 above)
Comment (ondrejhlavacek) [reply src/scripts/react/common/DestinationTableSelector.jsx]: detto

**Pull request merged**